.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -5,35 +5,27 @@ about: Report an issue or question while using nni instance (deployment).
 
 ---
 
-<!-- Please use this template while reporting an issue and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!-->
+**NNI Environment**:
+- NNI version:
+- NNI mode (local|remote|pai):
+- Client OS:
+- Server OS (for remote mode only):
+- Python version:
+- PyTorch/TensorFlow version:
+- Is conda/virtualenv/venv used?:
+- Is running in Docker?:
 
+**Log message**:
+ - nnimanager.log: 
+ - dispatcher.log:
+ - nnictl stdout and stderr:
+ 
+Where can you find the log files: [log](https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/HowToDebug.md#experiment-root-director), [stdout/stderr](https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/Nnictl.md#nnictl%20log%20stdout)
```

Comment:
Do NOT use hyperlink (`[x](xxx)`) in template file.

Reply:
OK. will change it to <!-- -->